packages/sanity/src/core/store/bundles/createBundlesStore.ts

```diff
@@ -171,7 +172,7 @@ export function createBundlesStore(context: {client: SanityClient | null}): Bund
         // After successful reconnection, fetch the list. Note that the first event is skipped, so
         // this will not occur upon initial connection.
         if (event.type === 'welcome') {
-          return listFetch$
+          return listFetch$.pipe(delay(1_000))
```

Comment:
Curious on why this change is needed

Reply:
This looks like part of a rebase that has gone wrong 🫠 
hold please...